packages/clarity-js/types/data.d.ts

```diff
@@ -134,7 +136,8 @@ export const enum BooleanFlag {
 
 export const enum Setting {
     Expire = 365, // 1 Year
-    SessionTimeout = 30 * Time.Minute, // 10 minutes
+    SessionExpire = 1, // 1 Day
+    SessionTimeout = 30 * Time.Minute, // 30 minutes
```

Comment:
Why shrink?

Reply:
Not expanding... only updating the comment to reflect the true value so comment stays accurate.